static/less/humhub.less

```diff
@@ -204,6 +204,11 @@
     @import "print.less";
 }
 
+@prev-directory: false;
+& when not(@prev-directory) {
```

Comment:
Would it also be enough to flag the include as deprecated with a comment?
FYI: The Less file is usually compiled when the package is created and the user does not normally have to do this.

Reply:
Sorry, it seems I cannot understand your question correctly.
I added the code in order to keep all new CSS for new directory(people and spaces) in the new file `directory.less`:

```css
@prev-directory: false;
& when not(@prev-directory) {
    @import "directory.less";
}
```

We should not deprecate this, it is not related to the old directory module.

Reply:
@yurabakhtin Sorry for the confusion, I misunderstood when flying throu the code.